### DISCUSSION TITLE
Other K-sizes ?

### DISCUSSION THREAD

**Discussion by zackclark70** [Chia-Network/bladebit#72] [Q&A] [answered]
it seems there may be a demand to have a full ram plotter for chives coin ( they run k29-k31 ) there is funds available for projects  it would seem that it would be great if k29 was possible with something like this as it would make ram plotting possible for everyone even with  64gb of ram the devs would be open to talking or i would be happy to deal with it

**Comment by tingyuloulala** [comment]
yes. it sounds exciting!

**Comment by harold-b** [comment, marked as answer]
There won't be any support below k32.

**Reply by zackclark70** [reply]
that is a big shame as the chives project is gaining traction now and the k29-k31 is running just fine what is the reason for this as i am sure funding could be gathered to pay for this

**Reply by harold-b** [reply]
That would be a pretty big conflict of interest, now wouldn't it? 😄 

Please see https://github.com/Chia-Network/bladebit/discussions/103

**Reply by zackclark70** [reply]
given that the whole project is open source it seems kind of petty to keep an energy efferent environmentally better plotter just to 1 project it goes against the whole green crypto movement

**Reply by justinengland** [reply]
I'm sorry but plot sizes lower than k32 just aren't secure. If this fork runs on these small plot sizes it's vulnerable to grind attack. As this plotter is open source. You are welcome to make your own fork and change it to handle smaller plot sizes.

**Reply by zackclark70** [reply]
that is not the whole truth as you change k size the simulated net space goes down so you can probably simulate  6tb with k29 but you would need 10-30x the power than a 6tb hdd so totally pointless
